### PR TITLE
SKUPPER-381 expose from other ns

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

**Comment by pwright** [comment]
@ajssmith could you take a look?

File: modules/cli/pages/index.adoc [hunk 1]
```diff
@@ -300,6 +300,44 @@ $ skupper service bind hello-world-backend deployment hello-world-backend
...
+* A service running in a different namespace.
+* A network policy that allows communication between the namespaces
+
+. Log into your cluster and change to the namespace for your site.
+
+. Expose the service on the {service-network}:
++
+--
+----
+$ skupper gateway expose service <service>.<namespace> --address <service>
```
Comment (ajssmith) [inline modules/cli/pages/index.adoc:321]: iiuc, you did not intend gateway here...

Comment (pwright) [reply modules/cli/pages/index.adoc]: doh!

**Review by ajssmith — APPROVED** [review]
lgtm

**Pull request merged**